dle/cfn/deploy_django.yml

```diff
@@ -0,0 +1,285 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  ElasticIPAddress:
+    Type: String
+    Default: "44.197.203.81"
```

Comment:
Is this created by Cloudfront? Otherwise we need to identify a list of things that aren't set up by CFN